test/byContract/IexecPocoBoost/IexecEscrow.v8.test.ts

```diff
@@ -10,7 +11,6 @@ import { getIexecAccounts } from '../../../utils/poco-tools';
 
 const userBalance = 1000;
```

Comment:
Maybe rename user everywhere to account to harmonize

Reply:
Done [Rename user by account](https://github.com/iExecBlockchainComputing/PoCo/pull/155/commits/e4eceba4f1e894e7c0a147100f90cc96ab01ff23)